plugin/index.js

```diff
@@ -1,13 +1,22 @@
 'use strict';
```

Comment:
I don't think we should commit output files to the repo, it should be somehow generated automatically (the same applies for `index.d.ts` and `index.js.map`).